### ISSUE TITLE
Exercise 13.3 "Library"

### ISSUE THREAD

**Issue by pdawyndt** [dodona-edu/programmeursleerling#175]
### checklist

- [ ] title (English)
- [ ] title (Dutch)
- [ ] unit tests
- [ ] review unit tests
- [ ] English
  - [ ] title
  - [ ] description
  - [ ] sample solution
  - [ ] review description
  - [ ] review sample solution
- [ ] Dutch
  - [ ] title
  - [ ] description
  - [ ] sample solution
  - [ ] review description
  - [ ] review sample solution

**Comment by TomNaessens** [comment]
🤖 PR #336 builds this exercise out. It was a stub: descriptions in both languages, but no `evaluation/` and no `solution/`, and parked as `type: content` so it would not surface as a broken exercise. It now has three tabs of doctests (`list2dict` / `location` / `titles`), reference solutions in both languages, and a generator.

Worth flagging for the tracker: `unit tests` and both `sample solution` boxes above were already ticked, but on `master` this exercise contains only `config.json`, the two descriptions and the two readmes. Those ticks claimed work that did not exist. After #336 they are accurate.

The five `review ...` items still need a human reviewer. Two judgement calls in there worth a look, since the book poses this one as an open design question rather than a specification: the API is invented, and the answer now sits in a spoiler under the question.
